ui/app/helpers/utils/fetch-with-cache.js

```diff
@@ -16,16 +16,15 @@ const fetchWithCache = async (
     fetchOptions.headers = new window.Headers(fetchOptions.headers)
   }
   if (
-    fetchOptions.headers &&
```

Comment:
This was removed because it was unnecessary, though it is unrelated to the other changes in this PR.